src/libraries/MultisigTypes.sol

```diff
@@ -89,3 +89,10 @@ struct TaskPayload {
     bytes[] calldatas;
     uint256[] originalNonces;
 }
+
+/// @notice Struct to store information about a safe
+struct SafeData {
+    address addr;
```

Comment:
```suggestion
    address safe;
```
Just IMO, I prefer not to use types as names.

Reply:
I'll address these comments in a later PR so resolving for now.